.github/workflows/docker-publish.yml

```diff
@@ -33,9 +33,9 @@ jobs:
             image: ghcr.io/teamhanko/hanko
           - context: ./quickstart
             image: ghcr.io/teamhanko/hanko/quickstart
-          - context: ./elements
+          - context: ./frontend
```

Comment:
Is it correct that here the bundled docker image with `elements` and `frontend-sdk` should be build and published and not the one which only contains the `element`?

If so, why do we still have a different image for `frontend-sdk`?

Reply:
Good question. We now have the elements docker image that includes the frontend-sdk. Would be sufficient for us to use just this image. Idk if anyone will use the frontend-sdk only. I would leave it for now as it doesn't do any harm and we could take an look at the donwload numbers in the future to determine if it's needed.

Reply:
👍